scripts/ci/build_dpdk.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash -x
+
+set -e
+
+DPDK_VERS="17.11.2"
```

Comment:
Ugh. This whole script should go into image building stage.
Also ideally we should be able to install DPDK into system location. Can you check if backported DPDK packages are available? Or maybe we can just backport and build them manually?

Reply:
apt-get has some dpdk packages. I build dpdk  with kernel headers from host. Docker use hosts kernel. Not sure if we can build dpdk inside Docker and run it on different hosts.

Reply:
dpdk in this version is also 17.11,  looking if odp will work with dpdk from apt..